commits.go

```diff
@@ -247,3 +250,103 @@ func (s *CommitsService) PostCommitComment(
 
 	return c, resp, err
 }
+
+// GetCommitStatusesOptions represents the available GetCommitStatuses() options.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/api/commits.html#get-the-status-of-a-commit
+type GetCommitStatusesOptions struct {
+	Ref   string `url:"ref,omitempty"`
+	Stage string `url:"stage,omitempty"`
+	Name  string `url:"name,omitempty"`
+	All   bool   `url:"all,omitempty"`
```

Comment:
All the xxxOptions struct now have both `url` and `json` tags. I know this is a `GET` API call, but to keep things consistent, please add the `json` tags here as well.